clang/lib/CodeGen/CodeGenFunction.cpp

```diff
@@ -1566,9 +1599,21 @@ void CodeGenFunction::GenerateCode(GlobalDecl GD, llvm::Function *Fn,
     }
   }
 
+  if (SanOpts.has(SanitizerKind::Realtime)) {
```

Comment:
This behaviour (and the exit points equivalent) would also be great to get under unit test if we can

Reply:
Check these tests out in clang/test/Driver